### PR TITLE
Upgrade libraries for aws-sdk-go , k8s 1.23, etc.

### PR DESCRIPTION
**Pull request by cruizen**
* aws-sdk-go https://github.com/aws/aws-sdk-go/releases has some updates to ec2, _**_eks_**_ etc.

### CODE REVIEW

**Comment by bcle** [comment]
Can you add a `size:` estimate label.

**Comment by bcle** [comment]
This isn't just a library version update. I see other changes. Can you summarize. Also, why exactly do we need this aws-sdk update? You mention ecs, eks, ... but why now?

**Comment by cruizen** [comment]
Closing this is now superseded by more recent PRs.
